pkg/storage/storage.go

```diff
@@ -54,6 +54,7 @@ type RelationshipTupleReader interface {
 
 	// ReadPage is similar to Read, but with PaginationOptions. Instead of returning a TupleIterator, ReadPage
 	// returns a page of tuples and a possibly non-empty continuation token.
+	// There is NO guarantee on the order returned on the list.
```

Comment:
I thought there is guarantee order on the tuples for ReadPage?  Both Postgres and MySQL quries with order by ulid.  If there is no order on the tuples, then continuation_token will not work.

Reply:
Ah you're right, my bad!